account_invoice_custom_rounding/models/account_move.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    tax_calculation_rounding_method = fields.Selection(
+        [
+            ("round_per_line", "Round per Line"),
+            ("round_globally", "Round Globally"),
+        ],
+        compute="_compute_tax_calculation_rounding_method",
+        store=True,
+        readonly=False,
+        help="How total tax amount is computed. If no value selected, "
+        "the method defined in the company is used.",
+    )
+
+    @api.depends("partner_id")
+    def _compute_tax_calculation_rounding_method(self):
+        for move in self:
+            tax_calculation_rounding_method = False
+            if (
+                move.move_type in ["out_invoice", "out_refund"]
```

Comment:
Why only out_invoices? Could be applied to in_invoices, isn't it?

I would recomend to do `move.is_invoice()`

Reply:
@etobella Thank you, we'll take a look.